docs/registries-authentication.md

```diff
@@ -0,0 +1,29 @@
+## Container Image Registries Authentication
+
+To authenticate with private container image registry you are required to provide
+[registry authentication file](https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md) to your podvm in order
+to allow the image to be pulled directly.
+
+Registry authentication file can be provided either statically or at runtime.
```

Comment:
@stevenhorsman anything to be mentioned specific to CC case and attestation ?

Reply:
I don't think so. At the moment we don't have proper CC support and I wouldn't want to indicate that it was, so I think we'll update the doc once we've firmed that up.